plugins/modules/make.py

```diff
@@ -83,7 +83,36 @@
     file: /some-project/Makefile
 '''
 
-RETURN = r'''# '''
+RETURN = r'''
+chdir:
+  description:
+    - Change to this directory before running make.
```

Comment:
Hmm, these are simply copies of the descriptions of the corresponding options, right? I think it would be better to say that these are simply the values passed in from the options:
```suggestion
    - The value of the module parameter I(chdir).
```
WDYT?